src/main/java/com/clertonleal/proto/Proto.java

```diff
@@ -98,8 +101,11 @@ private Proto() {}
                 }
             } while (cursor.moveToNext());
         }
-
-        if (closeCursor) {
+        if (closeCursor == null) {
```

Comment:
Ao invés disso, você pode manter o closeCursor primitivo, e passar o configuration().isClosingCursor() quando ele não for desejável como parâmetro.

Atual:
return list(cursor, clazz, null);

Depois: 
return list(cursor, clazz, configuration().isClosingCursor());

Isso iria evitar a checagem de 3 estados para o boolean. -> true, false, null